bin/dynamic_challenge_score.py

```diff
@@ -0,0 +1,437 @@
+#!/usr/bin/env python3
+
+import argparse
+import json
+import os
+import typing
+
+import tarfile
+import numpy as np
+from typing import Tuple, List
+import synapseclient
+
+
+def get_args():
+    """Set up command-line interface and get arguments without any flags."""
+    parser = argparse.ArgumentParser()
+    parser.add_argument("submission_id", type=str, help="The ID of submission")
+    parser.add_argument("status", type=str, help="The status of submission")
+    parser.add_argument(
+        "predictions_path", type=str, help="The path to the predictions folder"
+    )
+    parser.add_argument(
+        "groundtruth_path", type=str, help="The path to the ground truth folder"
+    )
+    parser.add_argument(
+        "output",
+        type=str,
+        nargs="?",
+        default="results.json",
+        help="The path to output file",
+    )
+
+    return parser.parse_args()
+
+
+# Since it's a data-to-model challenge, users will take care of taring their predictions locally
+def tar(directory: str, tar_filename: str) -> None:
+    """Tar all files in a directory without including the directory
+    Arguments:
+        directory: Directory path to files to tar
+        tar_filename:  tar file path
+    """
+    with tarfile.open(tar_filename, "w") as tar_o:
+        original_dir = os.getcwd()
+        os.chdir(directory)
+        for file in os.listdir("."):
+            tar_o.add(file, arcname=file)
+        os.chdir(original_dir)
+
+
+def untar(directory: str, tar_filename: str) -> None:
+    """Untar a tar file into a directory
+
+    Arguments:
+        directory: Path to directory to untar files
+        tar_filename:  tar file path
+    """
+    with tarfile.open(tar_filename, "r") as tar_o:
+        tar_o.extractall(path=directory)
```

Comment:
This scoring script seems outdated. Can you replace by [this one (V5)](https://www.synapse.org/#!Synapse:syn53601222)?

Reply:
Sorry about that, I didn't realize there was a 5th version

Reply:
Made these changes in my latest commit.